lib/cdo/geocoder.rb

```diff
@@ -60,6 +60,10 @@ def formatted_address
         data['place_name']
       end
 
+      def relevance
```

Comment:
add `relevance` to `Result::Base` above. If a developer doesn't have Mapbox configured, then it will default to a free service which might not have the `relevance` method. We want to avoid MethodNotFound errors. Or you could remove your local mapbox_access_token and see if it still works.

Maybe you could have it default to returning `1.0` or something...